fedscale/core/aggregator.py

```diff
@@ -40,9 +40,6 @@ def __init__(self, args):
         self.model_weights = collections.OrderedDict() # all weights including bias/#_batch_tracked (e.g., state_dict)
         self.last_gradient_weights = [] # only gradient variables
         self.model_state_dict = None
-        # NOTE: if <param_name, param_tensor> (e.g., model.parameters() in PyTorch), then False
-        # True, if <param_name, list_param_tensors> (e.g., layer.get_weights() in Tensorflow)
-        self.using_group_params = self.args.engine == events.TENSORFLOW 
```

Comment:
Please double check whether you have rebased to the master branch. You are removing our USEFUL code.

Reply:
@fanlai0990 Please let me know if this looks good.

Reply:
Thanks! It looks good now.